src/main/java/ch/njol/skript/expressions/ExprVectorRandom.java

```diff
@@ -36,12 +36,9 @@
 import ch.njol.util.Kleenean;
 import ch.njol.util.coll.CollectionUtils;
 
-/**
- * @author bi0qaw
- */
 @Name("Vectors - Random Vector")
-@Description("Creates a random vector.")
-@Examples({"set {_v} to a random vector"})
+@Description("Creates a random vector with the magnitude of 1.")
```

Comment:
```suggestion
@Description("Creates a random unit vector.")
```
Alternatively: `random normalized vector`.

Reply:
'unit vector' seems better. Thanks!